support/docs/usage.md

```diff
@@ -7,28 +7,36 @@ Usage
 namespace App\Email\WelcomeEmail;
 
 use FH\Bundle\MailerBundle\Composer\TemplatedEmailComposer;
+use FH\Bundle\MailerBundle\Composer\EmailComposer;
```

Comment:
Losse voorbeelden maken voor de verschillende composers. Zodat je het zo zou kunnen copy pasten in je project.